fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -261,6 +264,7 @@ def create_report2(
                     'thread_length_days': f'{thread_len:.1f}',
                     'thread_stddev_from_kw_avg_thread_len': '1+' if outlier else '0',
                     'thread_url': url,
+                    'query_date': today
```

Comment:
I think when you run this with the new column for the first time, what will happen is that any rows with new messages will show today's date. but anything that was previously cached and have no new messages will not show the query date. can you run it after you're done w/ it and see if that's the case?